docs/getting-started/roadmap_zh.md

```diff
@@ -4,27 +4,34 @@
 
 [milestones defined in GitHub](https://github.com/kubeedge/kubeedge/milestones) 包含了最新的开发进度
 
-KubeEdge 1.2 版本是我们目前的稳定版本， 在2019年Q4将添加如下的新特性。
+KubeEdge 1.3 版本是我们目前的稳定版本， 在2019年Q4将添加如下的新特性。
```

Comment:
I think maybe the 2019 Q4 should be 2020 Q2?

Reply:
Done.